config/api/src/main/java/org/jboss/windup/config/Variables.java

```diff
@@ -199,21 +201,24 @@ public void removeVariable(String name)
      * Searches the variables layers, top to bottom, for the iterable having all of it's items of the given type. Return
      * null if not found.
      */
-    public Iterable<? extends WindupVertexFrame> findVariableOfType(Class<?> type)
+    public <T> Iterable<T> findVariableOfType(Class<T> type)
```

Comment:
I would prefer:
public <T extends WindupVertexFrame> Iterable<T> findVariableOfType(Class<T> type)

Also, I'd prefer it to just return "return (Iterable<T>) frames;"